tests/test_dfes.py

```diff
@@ -573,13 +573,14 @@ def test_no_msprime_dfe(self):
         contig = stdpopsim.Contig.basic_contig(
             length=10000,
             mutation_rate=1e-6,
+            ploidy=2,
         )
         contig.add_dfe(
             intervals=np.array([[0, contig.length / 2]], dtype="int"),
             DFE=d,
         )
         model = stdpopsim.PiecewiseConstantSize(1000)
-        samples = model.get_samples(2)
+        samples = {"pop0": 1}
```

Comment:
Do we have a mix of conventions for population names: both `pop0` and `pop_0`? I know msprime introduces `pop_j` into population metadata for the j'th population, and SLiM uses `pj` for the j'th. Adding a third convention would be a shame!

Reply:
Oh good catch, it should be `pop_0` here. The error that this test checks gets raised before the misnomer gets checked.

Reply:
If the generic demographic models API gets reworked at some point (like if we transition to YAML as in #1232), we could make the naming consistent with SLiM.